tests/infra_test.go

```diff
@@ -128,7 +128,10 @@ var _ = Describe("[Serial][owner:@sig-compute]Infrastructure", func() {
 			secret.Data = map[string][]byte{
 				"random": []byte("nonsense"),
 			}
-			_, err = virtClient.CoreV1().Secrets(flags.KubeVirtInstallNamespace).Update(context.Background(), secret, metav1.UpdateOptions{})
+			err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
+				_, err = virtClient.CoreV1().Secrets(flags.KubeVirtInstallNamespace).Update(context.Background(), secret, metav1.UpdateOptions{})
```

Comment:
I don't think that this is better than before. You will also have to fetch the secret inside the `RetryOnConflict` function. Otherwise every retry will still try to PUT the secret with the outdated resource version and fail again.

Reply:
I missed that one, Thanks
secret fetch moved to `RetryOnConflict` block

Reply:
I think its an improvement, it will defiantly prevent conflict errors

> /lgtm
> /approve

Thank you!